pkg/testutil/rand/model.go

```diff
@@ -129,76 +129,80 @@ func WithMessageFormat() ModelMessageOption {
 	}
 }
 
-// ------------------Messages------------------
+// ------------------Translation-----------------
 
-// modelMessages generates a random model.Messages with the given
-// count of Messages.messages and using the provided options for each message.
-func modelMessages(msgCount uint, msgOpts ...ModelMessageOption) *model.Messages {
-	messages := &model.Messages{
+// modelTranslation generates a random model.Translation with the given
+// count of messages.message and using the provided options for each message.
+func modelTranslation(msgCount uint, msgOpts ...ModelMessageOption) *model.Translation {
+	translation := &model.Translation{
 		Language: Language(),
 		Original: gofakeit.Bool(),
 		Messages: make([]model.Message, msgCount),
 	}
 
 	if msgCount == 0 {
-		return messages
+		return translation
 	}
 
 	msgs := ModelMessageSlice(msgCount, msgOpts...)
 	for i, msg := range msgs {
-		messages.Messages[i] = *msg
+		translation.Messages[i] = *msg
 	}
 
-	return messages
+	return translation
 }
 
-// ModelMessages generates a random model.Messages with specific messages.message count, message and messages options.
-func ModelMessages(msgCount uint, msgOpts []ModelMessageOption, msgsOpts ...ModelMessagesOption) *model.Messages {
-	// msgsF wraps modelMessages() for mdl function.
-	msgsF := func() *model.Messages {
-		return modelMessages(msgCount, msgOpts...)
+// ModelTranslation generates a random model.Translation
+// with specific messages.message count, message and messages options.
+func ModelTranslation(msgCount uint,
+	msgOpts []ModelMessageOption,
+	translationOpts ...ModelTranslationOption,
+) *model.Translation {
+	// translationF wraps modelTranslation() for mdl function.
+	translationF := func() *model.Translation {
+		return modelTranslation(msgCount, msgOpts...)
 	}
 
-	return mdl(msgsF, msgsOpts...)
+	return mdl(translationF, translationOpts...)
 }
 
-// ModelMessagesSlice generates a slice of random model.Messages with the message and messages options.
-func ModelMessagesSlice(
+// ModelTranslations generates a slice of random model.Translation with the message and messages options.
```

Comment:
// ModelTranslations generates a slice of random model.Translation with the message and translation options.